codespell_lib/tests/test_dictionary.py

```diff
@@ -146,7 +156,7 @@ def test_error_checking(err, rep, match):
         _check_err_rep(err, rep, (None, None), 'dummy')
 
 
-@pytest.mark.skipif(speller is None, reason='requires aspell-en')
+@pytest.mark.skipif(not spellers, reason='requires aspell-en')
```

Comment:
Is this just all in aspell-en?

Reply:
The tests would only require the 'en' part of the dictionaries.
aspell-en contains: (for debian, but ubuntu should be the same)
`
en_AU.multi
en_AU-variant_0.multi
en_AU-variant_1.multi
en_AU-w_accents.multi
en_AU-wo_accents.multi
en_CA.multi
en_CA-variant_0.multi
en_CA-variant_1.multi
en_CA-w_accents.multi
en_CA-wo_accents.multi
en_GB-ise.multi
en_GB-ise-w_accents.multi
en_GB-ise-wo_accents.multi
en_GB-ize.multi
en_GB-ize-w_accents.multi
en_GB-ize-wo_accents.multi
en_GB.multi
en_GB-variant_0.multi
en_GB-variant_1.multi
en_GB-w_accents.multi
en_GB-wo_accents.multi
en.multi
en_US.multi
en_US-variant_0.multi
en_US-variant_1.multi
en_US-w_accents.multi
en_US-wo_accents.multi
en-variant_0.multi
en-variant_1.multi
en-variant_2.multi
en-w_accents.multi
en-wo_accents.multi
`